docs/resource-policy/installation.md

```diff
@@ -139,6 +139,14 @@ For the manual installation we will be using templating tool to generate Kuberne
 
 That's it! You have now installed the topology-aware NRI resource policy plugin using kutomize.
 
+Also, you can use the following **helm** command to install the NRI resource policy plugin in the **nri-plugins/deployment/helm/resource-management-policies/balloons** directory.
+
+```
+helm install balloons . --namespace kube-system
+```
+
+*Note: please specify the kube-system namespace.*
+
```

Comment:
I would instead put the `cd/pushd` into the directory as another step instead of having two helm install commands which are doing the same thing but from different directory.

Reply:
I think that we should provide the helm install and uninstall commands because people are familiar with them.

Reply:
Thanks for your link. I notice that there is a helm install command for the topology-aware policy. Why the balloon policy doesn't have any helm install command like topology-aware policy?

Reply:
Well the only thing that changes between plugins when installing via Helm is the directory and it is mentioned at the top of the document that we are showcasing TA as an example which can be applied to any other plugin hosted on this repo.

I don't think that it is a good idea to repeat the same commands for each plugin, especially when the number of the plugins
in the future. We expect that every plugin Helm charts follow the same structure in terms of file structure, thus ensure that same installation command can be applied to all plugins with minor changes of the directory and the chart name potentially.

Reply:
But perhaps we should make it a bit more clear that the same commands can be re-used for other plugins installations with minor changes?